pkg/operator/k8sutil/configmap_test.go

```diff
@@ -92,6 +92,7 @@ func TestGetOperatorSetting(t *testing.T) {
 	assert.Equal(t, envSettingValue, setting)
 
 	// ConfigMap is found
+	os.Setenv("POD_NAMESPACE", testNamespace)
```

Comment:
Was the unit test failing without this?

Reply:
@travisn Yes, it was. Previously we could get k8s resources through `k8s.io/client-go` with specifying ""(empty string) namespace. However, we can't do this now because `k8s.io/client-go` was modified to prevent that since `v1.17`.
(PreflightCheck logic was slightly modified: https://github.com/kubernetes/client-go/commit/e6a1dc4b133b01aacfdf0502a3ec504492233d17#diff-cce82935c007206a9e6070b09220908b)